addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/services/LogServiceImpl.java

```diff
@@ -25,6 +30,10 @@
 {
     private static final String LOG_DIR = "logs";
     private static final String LOG_FILENAME = "analysis.log";
+    private static final Set<String> WHITELIST_SET = Sets.newHashSet(
```

Comment:
I don't get where/how this is used

Reply:
Good catch. I had that in there earlier before changing the approach. It is no longer relevant and I have no removed it, along with the other unused constants.